extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -188,7 +188,7 @@ class AmpAccordion extends AMP.BaseElement {
       this.headers_.push(header);
 
       userAssert(
-        this.action_.hasAction(header, 'tap') == false,
+        this.action_.hasAction(header, 'tap', header.parentElement) == false,
```

Comment:
You can either use `section` or `dev().assertElement(header.parentElement)` to satisfy the type check (preference to `section` in my opinion).